core/src/test/scala/unit/kafka/api/ApiVersionTest.scala

```diff
@@ -184,6 +184,38 @@ class ApiVersionTest {
     assertEquals("3.2", KAFKA_3_2_IV0.shortVersion)
   }
 
+  @Test
```

Comment:
How about directly removing ApiVersion and ApiVersionTest? they are not used any longer